cmd/cedana-helper/install.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Check whether git is already installed
+if ! command -v git &> /dev/null; then
+    yum install -y git
```

Comment:
why yum? should we check the base OS and decide the package manager that way?

Reply:
Yes we should, we are only tested and working on the linux 2 image in eks so I didn't add support for anything else. But I can change this.

Reply:
OK awesome, just toss a comment in here for now then